tests/Aspire.Hosting.Azure.Tests/AzureContainerAppsTests.cs

```diff
@@ -98,7 +99,7 @@ param outputs_azure_container_apps_environment_id string
           }
         }
         """;
-
+        output.WriteLine(bicep);
```

Comment:
?

Reply:
You want this for when the bicep changes and you need to update it. You can copy and paste the new output from the test log